charts/vsphere-csi/values.yaml

```diff
@@ -208,6 +208,25 @@ controller:
     ## Enable debug mode
     ##
     debug: false
+
+  allcontainers:
```

Comment:
not to keen on that naming, i would usually prefer "global"
@mylesagray your opinion?

either way, i would like to stick to CamelCase here, so 'allContainers'

Reply:
Agree with @MaxRink on the use of `allContainers` - `global` is more universal

Reply:
@MaxRink @mylesagray Comment addressed. Please review again :pray: .